core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -170,16 +171,20 @@ public static LoadTableResponse stageTableCreate(Catalog catalog, Namespace name
     properties.putAll(request.properties());
 
     String location;
+    String metadataLocation;
     if (request.location() != null) {
       location = request.location();
+      metadataLocation = location + "/metadata";
     } else {
-      location = catalog.buildTable(ident, request.schema())
+      Table tbl = catalog.buildTable(ident, request.schema())
```

Comment:
In Iceberg, we don't avoid abbreviations like `tbl`. It's no easier to type and is harder to read. Use `table` instead.

Reply:
Changed to table.